rest/model/patch.go

```diff
@@ -39,7 +39,8 @@ type APIPatch struct {
 	GithubPatchData         githubPatch          `json:"github_patch_data,omitempty"`
 	ModuleCodeChanges       []APIModulePatch     `json:"module_code_changes"`
 	Parameters              []APIParameter       `json:"parameters"`
-	PatchedConfig           *string              `json:"patched_config"`
+	PatchedParserProject    *string              `json:"patched_config"`
+	PatchedProjectConfig    *string              `json:"patched_project_config"`
```

Comment:
Do we use this at all with APIPatches? Otherwise we shouldn't include it since it'll make routes like https://evergreen.mongodb.com/rest/v2/patches/61dc61bc0305b90e4255d28c crazier (I'm actually not sure if we should even be including PatchedParserProject but that's a different question)

Reply:
We don't - I'll remove it